docs/reference/docs/get.asciidoc

```diff
@@ -52,9 +52,8 @@ call in-place to make the document visible. This will also make other documents
 changed since the last refresh visible. In order to disable realtime GET,
 one can set the `realtime` parameter to `false`.
 
-When getting a document, one can specify `fields` to fetch from it. They
-will, when possible, be fetched as stored fields (fields mapped as
-<<mapping-store,stored>> in the mapping).
+When getting a document, one can specify `stored_fields` to fetch from it.
+They will be simply ignored if the field is not stored (<<mapping-store,stored>> in the mapping)*[]:
```

Comment:
Maybe just remove the whole paragraph because we already have the `=== Source filtering` section.